src/event.zig

```diff
@@ -3,5 +3,167 @@
 // SPDX-License-Identifier: MIT
 // This file is part of the Termelot project under the MIT license.
 
-pub const mouse = @import("event/mouse.zig");
-pub const key = @import("event/key.zig");
+const termelot = @import("../termelot.zig");
+const Rune = termelot.Rune;
+const Size = termelot.Size;
+
+/// A self-contained Event handler. The i64 parameter is for thread synchronization,
+/// and it is milliseconds since Unix Epoch. The callback function will be executed
+/// from a secondary thread when handling events. The time parameter is provided for
+/// synchronization. Don't forget about data races!
```

Comment:
```suggestion
/// synchronization. It is up to the user to prevent data races if the callback modifies
/// shared state through its context.
```

Clarify when user may be concerned about data races